code/model/modelread.cpp

```diff
@@ -3345,12 +3345,9 @@ int submodel_find_2d_bound_min(int model_num,int submodel, matrix *orient, vec3d
  */
 int subobj_find_2d_bound(float radius ,matrix * /*orient*/, vec3d * pos,int *x1, int *y1, int *x2, int *y2 )
```

Comment:
Please add a comment in this function's doxygen description (@details) to state how the function is working.  Something along the lines of "Uses a simple planar projection of the subsystem's radius to draw a targeting box around it" should suffice

Reply:
The function operates by calculating the focal length of the camera, then using the law of similar triangles to project the radius of the subsystem to the screen. I added a comment to this affect in the function documentation as per your request.